test/legacy/to-dep-tree.test.ts

```diff
@@ -210,11 +210,45 @@ describe('graphToDepTree with a linux pkgManager', () => {
   });
 });
 
-test('graphs with cycles are not supported', async () => {
-  const cyclicDepGraphData = helpers.loadFixture('cyclic-dep-graph.json');
-  const cyclicDepGraph = depGraphLib.createFromJSON(cyclicDepGraphData);
+describe('Cyclic graphs', () => {
+  test('cyclic graph to tree', async () => {
+    const cyclicDepGraphData = helpers.loadFixture('cyclic-dep-graph.json');
+    const cyclicDepGraph = depGraphLib.createFromJSON(cyclicDepGraphData);
+
+    const depTree = await depGraphLib.legacy.graphToDepTree(
+      cyclicDepGraph,
+      'pip',
+    );
+
+    const cyclicTreeNode = _.get(
```

Comment:
we should be able to use optional chaining now (instead of locrap)

Reply:
@aviadhahami You need to address this to get the tests to pass

Reply:
yup, on it
will take care post FE weekly :+1:

Reply:
I do like the name `locrap` thot
Fixed :D